lib/tasks/rummager.rake

```diff
@@ -45,6 +45,25 @@ namespace :rummager do
       end
       puts "Complete."
     end
+
+    desc "indexes all content belonging to a model"
```

Comment:
Not sure how it works, but it seems this is doing a closely related thing:

https://github.com/alphagov/whitehall/blob/bc91c35e8ad9e6a58968b25154f6e35d0ff54437/lib/tasks/rummager.rake#L26-L30

Could we re-use that, especially the batching stuff?